docs/source/installation.rst

```diff
@@ -48,46 +49,132 @@ You're done! Now go to:
 .. _docker: https://www.docker.io/
 
 
-MozDef manual installation process on RedHat systems
-----------------------------------------------------
+Manual Docker Installation
```

Comment:
This block doesn't have anything to do with Docker, does it make more sense to name it "Manual Installation"?